ifrs17/Import/Importers.ipynb

```diff
@@ -1458,6 +1458,7 @@
                 "\n        }, ImportFormats.Cashflow",
                 "\n    ).WithTarget(workspace).ExecuteAsync();",
                 "\n    ",
+                "\n    await workspace.CheckForCompositeAmountTypeAndNormalize(AmountTypes.CU);",
```

Comment:
give the storage as input. The storage should provide all it is needed (hierarchyCache)

Reply:
implemented